pl-csb-model/__main__.py

```diff
@@ -1,28 +1,44 @@
 from transformers import NllbTokenizer, AutoModelForSeq2SeqLM
 import argparse
 
+import sacrebleu
 import config_loader
 import train.data_loader as data_loader
 import train.model_finetuner as model_finetuner
+import evaluate.model_evaluator as model_evaluator
 import translate.translator as translator
 
 def train_model(config: dict) -> None:
-    pretrained_model = AutoModelForSeq2SeqLM.from_pretrained(config["MODEL"]["PretrainedModelName"])
-    train_data = data_loader.load_train()
-    tokenizer = NllbTokenizer.from_pretrained(config["MODEL"]["PretrainedModelName"], additional_special_tokens=["csb_Latn"])
+    pretrained_model = AutoModelForSeq2SeqLM.from_pretrained(config["MODEL"]["pretrained_model_name"])
+    tokenizer = NllbTokenizer.from_pretrained(config["MODEL"]["pretrained_model_name"], additional_special_tokens=["csb_Latn"])
+    train_data = data_loader.load_data(config["DATA"]["training_data_file"])
 
-    model_finetuner.finetune(pretrained_model, train_data, tokenizer, config)
+    model_finetuner.finetune(pretrained_model, tokenizer, train_data, config)
     
 def use_model(config: dict) -> None:
-    pretrained_model = AutoModelForSeq2SeqLM.from_pretrained(config["MODEL"]["OutputModelName"])
-    tokenizer = NllbTokenizer.from_pretrained(config["MODEL"]["OutputModelName"])
+    pretrained_model = AutoModelForSeq2SeqLM.from_pretrained(config["MODEL"]["output_model_name"])
+    tokenizer = NllbTokenizer.from_pretrained(config["MODEL"]["output_model_name"])
     message="Wsiądźmy do tego autobusu"
     translated_message=translator.translate(message, pretrained_model, tokenizer, 'pol_Latn', 'csb_Latn')
     print(f"Message {message} has been translated to: {translated_message}")
 
+def evaluate_model(config: dict) -> None:
+    pretrained_model = AutoModelForSeq2SeqLM.from_pretrained(config["MODEL"]["output_model_name"])
+    tokenizer = NllbTokenizer.from_pretrained(config["MODEL"]["output_model_name"])
+    eval_data = data_loader.load_data(config["DATA"]["evaluation_data_file"])
+
+    result = model_evaluator.evaluate(pretrained_model, tokenizer, eval_data)
+    print(f"BLEU Score ({eval_data.columns[0]} -> {eval_data.columns[1]}): {result.score}")
+
+    # Swap columns and evaluate model translating in the other direction
```

Comment:
Instead of writing this comment, create a helper function `evaluate_translations(model: AutoModelForSeq2SeqLM, tokenizer: NllbTokenizer, source: pd.DataFrame, target: pd.DataFrame) -> None`. This function should take a column in one language as `source_lang` and a column in another language as `target_lang` (I can see that you extract the columns in `evaluate` anyways). The target code is:
```python
polish_column = eval_data.columns[0]
kashubian_column = eval_data.columns[1]

evaluate_translations(pretrained_model, tokenizer, source=polish_column, target=kashubian_column)
evaluate_translations(pretrained_model, tokenizer, source=kashubian_column, target=polish_column)
```
This approach eliminates the need to hackishly swap columns and reduces code redundancy.